sections/placeholders.md

```diff
@@ -0,0 +1,117 @@
+
+##### Parameters not exposed through the API
+
+In the application's user interface, there are certain options available when creating and editing placeholder resources. For example, if a role and discipline are selected, the placeholder title is autopopulated as a combination of discipline and role. Options also exist to upload a thumbnail image for the placeholder, or to choose a color and abbreviation and then auto-generate a thumbnail. These options are currently not available through the API, and can only be used via the front-end UX.
+
+## Update a Placeholder
+
+Placeholder specified by `id`
+
+```
+PUT  /api/v1/placeholder_resources/<placeholder_resource_id>
+ curl -XPUT -d 'title=Senior Designer' \
+             'https://vnext.10000ft.com/api/v1/placeholder_resources/12345?auth=...'
+```
+
+## Delete a Placeholder
+
+Placeholder resources may be deleted via the API, as follows.
+
+```
+DELETE  /api/v1/placeholder_resources/<placeholder_resource_id>
+ curl -XDELETE \
+             'https://vnext.10000ft.com/api/v1/placeholder_resources/12345?auth=...'
+```
+
+## Placeholders and Users/People
+
+As noted above, placeholders behave similarly to real users. However, there are some differences. Here we discuss ways in which placeholders can be used similar to users in the API, and ways they are different.
+
+##### Placeholder Assignments
+
+Assignments can be made, updated, and removed via the [Assignments](sections/assignments.md) API just as they can for users. 
+The only difference is that this is accessed through the placeholder_resources endpoint. In general, for the [Assignments](sections/assignments.md) API, a placeholder resource ID can be used as the user_id parameter.
+
+```
+GET  /api/v1/placeholder_resources/<placeholder_resource_id>/assignments
+ curl 'https://vnext.10000ft.com/api/v1/placeholder_resources/12345/assignments?&auth=...'
+```
+
+##### Placeholder Bill Rates
+
+If a placeholder is given a role/discipline, they will inherit account default bill rates for that role/discipline. Specific bill rates can also be created for placeholders using the [bill rates](sections/bill-rates.md) API. A placeholder ID can be specified as the user_id parameter. No additional parameters are required.
+
+##### Placeholder Custom Fields
+
+[Custom fields](sections/custom-fields.md) can be created for placeholders through the application UI. API docs coming soon.
+
+##### Not Supported: Time Entries, Expense Items and Tags
+
+Time entries are not supported for placeholders, either through application front-end or via API. When a placeholder is assigned to a project, we will generated scheduled hours based on the type of assignment and update budget projections, so that placeholders can be used for forecasting/projection. However, placeholders do not incur time, and no explicit time entry changes can be made for them. Expense items are similarly not available for placeholders.
```

Comment:
> When a placeholder is assigned to a project, we will generated scheduled hours based

generated -> generate

Also, maybe consider wording this, "the system will generate."